plutus-core/plutus-core/src/PlutusCore/Default/Builtins.hs

```diff
@@ -119,6 +122,19 @@ data DefaultFun
     | MkPairData
     | MkNilData
     | MkNilPairData
+    -- Bitwise
+    | IntegerToByteString
+    | ByteStringToInteger
+    | AndByteString
+    | IorByteString
```

Comment:
This seems wrong? Shouldn't it be OrByteString? It is true that the CIP also says IOR for some reason, but all references I've been able to find to this term which I haven't heard of before explain it as _integer_ or-operation, as in, you give it to integers and it interprets them as bitstrings then does an or-operation on it, but this is an operation on bytestrings, not integers. AndByteString isn't named IAndByteString either, and neither are _any_ of the other operations.

Reply:
My logic was based on assembly language: typically you distinguish inclusive-or (denoted IOR) from exclusive-or (denoted XOR). Not particularly wedded to this though.

Reply:
FWIW the opcode is called OR in x86 AFAICT.

Reply:
[Indeed](https://www.felixcloutier.com/x86/or). As I said, not wedded to the idea.